melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/common/graphql/response/SafResponse.java

```diff
@@ -0,0 +1,9 @@
+package no.nav.melosys.eessi.integration.common.graphql.response;
+
+import lombok.Data;
+import no.nav.melosys.eessi.integration.saf.dto.SafJournalpost;
+
+@Data
+public class SafResponse {
```

Comment:
Hvorfor ligger det objektet i no.nav.melosys.eessi.integration.**common** når det er spesifikt for SAF?

Reply:
Fordi det ble litt refaktorering-kaos 😅  Nice catch, fikser